build/action.yml

```diff
@@ -9,6 +9,12 @@ inputs:
     description: 'The version of Python to use in the CI'
     required: true
     default: '3.x'
+  package-prefix:
+    description: |
+      The prefix (or name) of your pacakge (if applicable) to use
+      for GitHub releases
+    required: true
```

Comment:
will't "required: true" break existing workflows, which don't specify the package-prefix?

Reply:
No, it should be fine since it provides a default.  In this case, it's almost not necessary (similar to the Python version argument), but does serve as an indication that it's value is important to functionality and not any kind of add on behavior.